rebar.config

```diff
@@ -7,7 +7,7 @@
            , {d, 'APPLICATION', avlizer}
            , {d, 'NOTEST'}]}.
 
-{deps, [{erlavro, "2.9.0"}]}.
+{deps, [{erlavro, "2.9.8"}]}.
```

Comment:
This will update `rebar.lock` so please include that update as well (and squash into this commit).